### PR TITLE
[Core] Remove fluent  interfaces and add missing doc blocks

### PR DESCRIPTION
**Pull request by HerbMat**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Fixed tickets | no 
| License       | MIT

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Checkout/Step/PaymentStep.php [hunk 1]
```diff
@@ -72,6 +81,11 @@ protected function renderStep(ProcessContextInterface $context, OrderInterface $
         ));
     }
 
+    /**
+     * @param OrderInterface $order
+     *
+     * @return Form
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CoreBundle/Checkout/Step/PaymentStep.php]: `@return FormInterface`

File: src/Sylius/Bundle/CoreBundle/Checkout/Step/ShippingStep.php [hunk 2]
```diff
@@ -84,6 +93,11 @@ protected function renderStep(ProcessContextInterface $context, OrderInterface $
         ));
     }
 
+    /**
+     * @param OrderInterface $order
+     *
+     * @return Form
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CoreBundle/Checkout/Step/ShippingStep.php]: `@return FormInterface`

File: src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php [hunk 3]
```diff
@@ -284,6 +285,13 @@ public function countByCustomerAndPaymentState(CustomerInterface $customer, $sta
         ;
     }
 
+    /**
+     * @param \DateTime $from
+     * @param \DateTime $to
+     * @param string    $state
+     *
+     * @return array
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php]: `@return OrderInterface[]`

**Comment by pjedrzejewski** [comment]
Hey @HerbMat, could you rebase this PR? Thank you! :)

**Comment by okwinza** [comment]
@pjedrzejewski this seems abandoned. Do you want me to pick this up?

**Comment by pjedrzejewski** [comment]
@okwinza Sure thing! :)

**Comment by michalmarcinkowski** [comment]
This was replaced by https://github.com/Sylius/Sylius/pull/4862.

**Pull request closed without merging**